### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by DanielRosenwasser** [microsoft/TypeScript-Website#2371]
We have an internal accessibility team at Microsoft which has been able to point out the following issues

* [ ] TypeScriptLang.org_Community_AI4W: "H1" Heading level tag is not defined in the page.
* [ ] TypeScriptLang.org_Index_AI4W:Ensures aria-hidden elements do not contain focusable elements (`#playground-samples-popover`)
* [ ] TypeScriptLang.org_Tools_Playground_AI4W: Ensures that lists are structured correctly (`#compiler-options-dropdown`)
* [ ] TypeScriptLang.org_Tools_Playground_AI4W: Ensures elements with an ARIA role that require child roles contain them (`ol[role="tablist"]`)
* [ ] TypeScriptLang.org_Tools_Playground_AI4W: Ensures elements with an ARIA role that require parent roles are contained by them (`.selected.section-name.button`)
* [ ] TypeScriptLang.org_Tools_Playground_AI4W: Ensures every id attribute value of active elements is unique (`div:nth-child(1)`)
* [ ] TypeScriptLang.org_Tools_Playground_AI4W: Ensures `<li>` elements are used semantically (`.active:nth-child(8)`)
* [ ] TypeScriptLang.org_Documentation_Get Started_AI4W: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (`span:nth-child(3)`)
* [ ] TypeScriptLang.org_Documentation_Handbook_AI4W: Heading level tag is incorrectly defined in the page.
* [ ] TypeScriptLang.org_Documentation_Handbook_Keyboard: on High Contrast mode the focus is not visible on which topic we are under "On this Page"
* [ ] TypeScriptLang.org_Documentation_Handbook_Focus Order: Keyboard Focus is going on "like this page" which is incorrect focus order
* [ ] TypeScriptLang.org_Documentation_Handbook_Focus Visible: Keyboard Focus is not visible when it reaches to "like the page" and "dislike the page"
* [ ] TypeScriptLang.org-Web-Update Accessibility Conformance Documentation
* [ ] TypeScript website _Index_Reflow: In reflow some controls are missing in the navigation bar(Download, Documentation, Handbook, Playground)

**Comment by orta** [comment]
>  TypeScript website _Index_Reflow: In reflow some controls are missing in the navigation bar(Download, Documentation, Handbook, Playground) 

Is intentional those links are still accessible through other methods, they're just available quicker when there's space
